Error: Navigation lists must be nested in a `nav` element
Error: Current page must be indicated using `aria-current="page"`
Error: The current page is Products, but `aria-current` was applied to Home
Error: Styles were not requested
Error: Style changes (such as bold) must not be the only means of indicating a link is the current page
diff --git a/output/Basic Navigation with Current Page/1/8f341efa-fc8d-473b-9bee-c1d369378c30.html b/output/Basic Navigation with Current Page/1/8f341efa-fc8d-473b-9bee-c1d369378c30.html
index 1db3aa9..50e0a3d 100644
--- a/output/Basic Navigation with Current Page/1/8f341efa-fc8d-473b-9bee-c1d369378c30.html	
+++ b/output/Basic Navigation with Current Page/1/8f341efa-fc8d-473b-9bee-c1d369378c30.html	
@@ -1,18 +1,12 @@
-<ul>
-  <li><a href="/" aria-current="page">Home</a></li>
-  <li><a href="/products" class="current">Products</a>
-    <ul>
-      <li><a href="/products/frobs">Frobs</a></li>
-      <li><a href="/products/thrips">Thrips</a></li>
-    </ul>
-  </li>
-  <li><a href="/services">Services</a></li>
-</ul>
-<style>
-  .current {
-    font-weight: bold;
-  }
-  [aria-current="page"] {
-    color: red;
-  }
-</style>
\ No newline at end of file
+<nav>
+  <ul>
+    <li><a href="/">Home</a></li>
+    <li><a href="/products" class="current" aria-current="page">Products</a>
+      <ul>
+        <li><a href="/products/frobs">Frobs</a></li>
+        <li><a href="/products/thrips">Thrips</a></li>
+      </ul>
+    </li>
+    <li><a href="/services">Services</a></li>
+  </ul>
+</nav>
\ No newline at end of file
